apps/dashboard/app/javascript/script_edit.js

```diff
@@ -148,38 +148,82 @@ function addInProgressField(event) {
   enableNewFieldButton();
 }
 
+function fixExcludeBasedOnSelect(selectElement) {
+  const excludeElementId = selectElement.dataset.excludeId;
```

Comment:
I see this optimization of removing the `target-id` and using this `exclude-id`. 

But it now seems we can take that further and just use `${selectElement.id}_exclude` without having to embed the `data-exlclude-id` at all?

Reply:
I prefer it explicit in the template. It shows that is is important and is used for something else.
But happy to remove it and use the implicit id by using the format as you describe it.

Let me know if you prefer the optimization.

Reply:
>I prefer it explicit in the template. It shows that is is important and is used for something else.

If you have this preference, that's fine we can keep it as is.